doc/news/changes/major/20171204MartinKronbichler

```diff
@@ -3,9 +3,8 @@ MappingQGeneric have been changed from the old Laplace-style smoothing to a
 transfinite interpolation that linearly blends between the descriptions on the
 faces around a cell. The old transformation introduced boundary layers inside
 cells that prevented convergence rates to exceed 3.5 in the global L2 errors
```

Comment:
should be 'from exceeding 3.5'

Reply:
Thanks for pointing this out. I'm always happy to learn (or rather repeat) these subtleties of the English language... :+1:

Reply:
👍